src/pages/Home.mkpe1.js

```diff
@@ -3,14 +3,45 @@
 import { authentication } from 'wix-members';
 import { currentMember } from 'wix-members';
 
+import {
+    utils_set_sections_history,
+    utils_config_items,
+    SECTION_STATE_LOADING,
+    SECTION_STATE_DATA,
+    SECTION_STATE_NO_DATA,
+} from 'public/utils';
+
+import {
+    be_mod_utils_get_history,
+} from 'backend/be_mod_utils';
+
+const QTDE_ITENS_RESUMO = 2; // max number of transactions on resume
+
+
 async function fill_member_picture() {
 	let member = await currentMember.getMember({fieldsets: [ 'FULL' ]});
 	if (member.profile.profilePhoto.url)
 		$w("#imageProfile").src = member.profile.profilePhoto.url;
 	$w("#textOlaNome").text = `Olá, ${member.contactDetails.firstName}`;
 }
 
+async function load_resumed_history() {
+    $w("#repeaterHist").onItemReady( ($item, itemData, index) => {
+        utils_config_items($item, itemData)
+        utils_set_sections_history(SECTION_STATE_DATA);
+    });
+    
+    let history = (await be_mod_utils_get_history()).reverse();
+    history = history.length >= QTDE_ITENS_RESUMO ? history.slice(0 , QTDE_ITENS_RESUMO) : history; // limits items on resume
```

Comment:
I think you can send to the backend a parameter of how much you want to load, so you don't need to load everything and it would be lighter

Reply:
This topic was added on TODO board list because wix has a function to be used to limits data filter. I did not found something like that on js (I supposed to use a 'for', but it's not necessary w/ wix).

When change from json test to database this change will be added.